tinybuild-arm/Dockerfile

```
@@ -0,0 +1,11 @@
+ARG DOCKER_REGISTRY="docker.io/riot"
+FROM ${DOCKER_REGISTRY}/tinybuild-base:latest
```

```suggestion
ARG DOCKER_REGISTRY="docker.io/riot"
FROM ${DOCKER_REGISTRY}/tinybuild-apks:latest AS tinybuild-apks
FROM ${DOCKER_REGISTRY}/tinybuild-base:latest
```

Same here and for the other Dockerfiles.

---

I think I ran that locally with `docker buildx build . -t tinybuild-apks` and never gave it a docker registry prefix, as it was never intended to be pushed anywhere. I think this also how the CI script builds them.

---

The issue is that without a prefix, the `--mount=type=bind,target=/tinybuild-apks,source=/output,from=tinybuild-apks` tries to fetch it from `docker.io`.

But yeah, when the `tinybuild-apks` is built without the `local/` prefix, it works without my suggested modification.
That means you can revert the other change too 😅 


This is how I called it now:
```
riotdocker$ docker buildx build ./tinybuild-apks -t tinybuild-apks:latest

riotdocker$ docker buildx build --build-arg DOCKER_REGISTRY="local" ./tinybuild-base/ -t local/tinybuild-base:latest

riotdocker$ docker buildx build --build-arg DOCKER_REGISTRY="local" ./tinybuild-arm/ -t local/tinybuild-arm:latest

buechse@skyleaf:~/RIOTstuff/riotdocker-tinybuild/riotdocker$ docker image list
REPOSITORY                           TAG                       IMAGE ID       CREATED             SIZE
local/tinybuild-arm                  latest                    0409dc0bcba2   56 minutes ago      597MB
local/tinybuild-base                 latest                    e2e85be9c333   57 minutes ago      62.1MB
tinybuild-apks                       latest                    fc5f849f1da5   About an hour ago   351MB
```